cmd/helm/verify_test.go

```diff
@@ -24,13 +24,13 @@ import (
 
 func TestVerifyCmd(t *testing.T) {
 
-	stat_exe := "stat"
-	stat_path_msg := "no such file or directory"
-	stat_file_msg := stat_path_msg
+	statExe := "stat"
+	statPathMsg := "no such file or directory"
+	statFileMsg := statPathMsg
 	if runtime.GOOS == "windows" {
-		stat_exe = "GetFileAttributesEx"
-		stat_path_msg = "The system cannot find the path specified."
-		stat_file_msg = "The system cannot find the file specified."
+		statExe = "GetFileAttributesEx"
+		statPathMsg = "The system cannot find the path specified."
```

Comment:
I looked and can't find these. It appears that in syscall it's just the error codes that are exported.